admin_manual/office/migration.rst

```diff
@@ -0,0 +1,39 @@
+===============================
+Migration from Collabora Online
+===============================
+
+Nextcloud Office is based on Collabora Online so for enabling all Nextcloud Office functionallity it would be enough to update to the most recent release. Nextcloud Office is available since CODE 21.11.
```

Comment:
Typo.

```suggestion
Nextcloud Office is based on Collabora Online so for enabling all Nextcloud Office functionality it would be enough to update to the most recent release. Nextcloud Office is available since CODE 21.11.
```